app/components/modals/modal-base.js

```diff
@@ -1,8 +1,9 @@
 import Ember from 'ember';
+import UiModel from 'semantic-ui-ember/components/ui-modal';
```

Comment:
@niranjan94 shouldn't this be `UiModal` , model seems mis leading.
Same in the commit message too.